ansible/inventory/env/group_vars/all.yml

```diff
@@ -342,7 +342,7 @@ sunbird_device_register_api: "{{proto}}://{{api_proxy_name}}/v3/device/register/
 sunbird_course_batch_notification_signature: sunbird
 
 sunbird_authorization: "{{core_vault_sunbird_api_auth_token}}"
-sunbird_lms_base_url: "http://{{sunbird_swarm_manager_lb_ip}}:8000"
+sunbird_lms_base_url: "http://lms-service.{{namespace}}.svc.cluster.local:9000"
```

Comment:
Port 8000 is kong. Please confirm once in some env if this is kong or lms.

Reply:
this was the lb config for swarm. 8000 -> 9000, because we had one internal lb for all routing

Reply:
I don’t think so. We have /api at the end of the variable in some places.